.identity/02_application_master.tf

```diff
@@ -0,0 +1,28 @@
+resource "azuread_application" "master" {
```

Comment:
is there a way to encapsulate all these files into a terraform module in order to leave here only the parameters that have to be configured? (ie. the branch name and so on)

Reply:
yes we can, but I want to think about it... It's not complex or verbose configuration